### PR TITLE
Feature/filter by category

### PR DESCRIPTION
**Pull request by derekmiranda**
addresses https://github.com/sunrisemvmtnyc/legislation-tracker/issues/39

Filters bills by category on backend
- consolidates filtering within API
- however, the filter only extends to the initial bills within the `limit` number so may have to make additional bill requests to ensure we gather all bills after `limit` that match the filtered categories

Currently, no bills in the first 100 are in the categories so added a temp entry ('A2229' w/ 'renews' category)

### CODE REVIEW

**Comment by agreffard** [comment]
Sorry I took long to read the PR and respond.

From what I remember, we said we would only display the Sunrise featured bills ayway, right?
If that's the case, maybe we could be more specific with the elastic search to directly fetch only the list of bills we want (coming from the Airtable) :

https://legislation.nysenate.gov/api/3/bills/2023/search?term=printNo:(S3545 OR S4024 OR ...)&key=[API KEY]

it means we could also get rid of the pagination/limit issues

We can talk about it tomorrow! @derekmiranda @elliott-king @twblackwell @afnanhaq 
(We have multiple debates about it o I might be wrong)

**Review by elliott-king — DISMISSED** [review]
The other PR seems blocked on this. Let's just merge it, and I'll make a new PR if necessary, instead of delaying you.

**Comment by derekmiranda** [comment]
> The other PR seems blocked on this. Let's just merge it, and I'll make a new PR if necessary, instead of delaying you.

@elliott-king just need you to re-approve this PR since i made a change after you approved. feel free to merge it as well

**Review by elliott-king — APPROVED** [review]

**Pull request merged**
